scaleway/table_scaleway_billing_consumption.go

```diff
@@ -0,0 +1,141 @@
+package scaleway
+
+import (
+	"context"
+	"fmt"
+
+	billing "github.com/scaleway/scaleway-sdk-go/api/billing/v2beta1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableScalewayBillingConsumption(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "scaleway_billing_consumption",
+		Description: "A Scaleway Billing Consumption.",
+		List: &plugin.ListConfig{
+			Hydrate: listBillingConsumption,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "product_name",
+					Require: plugin.Optional,
```

Comment:
The code to handle the optional qual seems to be missing.

Reply:
i had this :)